data/spec_adopters.yml

```diff
@@ -97,7 +97,16 @@ adopters:
       homepage_url: https://twitter.com/TurkeyJBUG/
       logo: turkeyjboss.png
       logo_white: turkeyjboss.png
-      
+
+ - name: Jakarta NoSQL
```

Comment:
The joys of YAML. You need to fix the indentation here, so it is aligned with the others.

Reply:
Hi Ivar:

Thanks for reviewing and finding these errors.  I could have sworn I had everything lined-up nicely in IntelliJ IDEA.  Anyway, I re-cloned the repository and started fresh.  I also didn't use IDEA this time - sometimes vi is the best choice!  :-)

I will be submitting a new pull request...

Mike.